src/OrderFormAcceptanceTests.Steps/Steps/AdditionalServices.cs

```diff
@@ -335,7 +335,7 @@ public void GivenTheSelectAdditionalServiceFormIsPresented()
             Test.Pages.OrderForm.ClickAddSolutionButton();
             Test.Pages.OrderForm.EditAdditionalServicesSectionDisplayed().Should().BeTrue();
         }
-
+        [StepDefinition(@"the User selects an Associated Service with only one list price")]
```

Comment:
There needs to be a blank line between the end of a method and the first step definition of the next method

Reply:
sorting

Reply:
Done